run_tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+script_dir=$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd -P)
+
+${script_dir}/tests/harness.sh --test ${script_dir}/tests/default_options.conf --expected ${script_dir}/tests/default_options_expected.txt
```

Comment:
```suggestion
${script_dir}/tests/harness.sh --test ${script_dir}/tests/default_options.conf --expected ${script_dir}/tests/default_options_expected.txt "$@"
```

Allows `--no-color`, overwriting expected output, etc
I guess the only practical use case for this is `--no-color` when adding more tests, which is already possible via env var.

Reply:
Ideally `--no-color` would be determined automatically (I've confirmed this works as intended with github actions and my terminal). Also not a bad idea to let this be overridden if required (or any other options added in the future)